README.md

```diff
@@ -137,3 +137,33 @@ const browser = await launch();
 // Connect to first browser instead
 const anotherBrowser = await launch({ wsEndpoint: browser.wsEndpoint() });
 ```
+
+## BYOB - Bring Your Own Browser
+
+
+Essentially the process is as simple as running a chromium-like binary with the following flags:
+
+```
+chromium --remote-debugging-port=<SOME_PORT> \
+--headless=new \
```

Comment:
Just noting that `--headless=new` will change in the future, but this is fine for now.

Reply:
Probably fine to say nothing, just a note for someone in the future if this stops working.